db/migrate/20201109093337_create_trucks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTrucks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :trucks do |t|
+      t.string :license_plate
+      t.integer :capacity
+      t.references :model_type
```

Comment:
Is there a `model_types` table present?

Reply:
not yet, we will be adding that in a future PR.

Reply:
Maybe add a reference with the table creation?

Reply:
i believe so, i havent added it in